inc/functions-admin.php

```diff
@@ -5,6 +5,26 @@
 
 function tna_homepage_menu() {
 	add_menu_page( 'Homepage settings', 'Homepage', 'administrator', 'homepage-admin-page', 'homepage_admin_page', 'dashicons-admin-generic', 21  );
+
+	add_action( 'admin_init', 'homepage_admin_page_settings' );
+}
+
+function homepage_admin_page_settings() {
+	register_setting( 'homepage-settings-group', 'home_masthead_descr' );
+	register_setting( 'homepage-settings-group', 'home_masthead_btn_text_1' );
+
+	register_setting( 'homepage-settings-group', 'home_landing_page_title_1' );
```

Comment:
The register setting seems repetitive. I'm happy with the code, but if there are further requirements in the future to add more, i would suggest using a loop, which will keep the code DRY.